model/src/main/kotlin/MergedAnalyzerResult.kt

```diff
@@ -22,6 +22,7 @@ package com.here.ort.model
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import java.io.File
```

Comment:
Nit: This should be sorted next to the block below.

Reply:
Done.